### PR TITLE
feat(ACMG): Added activation strength to ACMG classification

### PR DESCRIPTION
**Pull request by Antonior26**
This solves CIPAPI-311. As requested by cardioClassifer team.

### CODE REVIEW

File: schemas/IDLs/org.gel.models.report.avro/6.1.0/CommonInterpreted.avdl [hunk 1]
```diff
@@ -215,6 +215,15 @@ enum TimeUnit {years, months, weeks, days, hours, minutes, na}
...
+    /**
+    Activation Strength enumertation:
+* `strong`
+* `moderate`
+* `supporting`
+    */
+    enum ActivationStrength{strong, moderate, supporting}
+
     /**
     Type of evidence in tge AMP guideline
```
Comment (cboustred) [inline schemas/IDLs/org.gel.models.report.avro/6.1.0/CommonInterpreted.avdl]: typo 'tge'

File: schemas/IDLs/org.gel.models.report.avro/6.1.0/CommonInterpreted.avdl [hunk 2]
```diff
@@ -262,6 +271,10 @@ enum TimeUnit {years, months, weeks, days, hours, minutes, na}
         */
         int modifier;
         /**
+        Activation Strength
+        */
+        ActivationStrength  activationStrength;
```
Comment (cboustred) [inline schemas/IDLs/org.gel.models.report.avro/6.1.0/CommonInterpreted.avdl]: should this be:

`ActivationStrength ActivationStrength;`

not

`ActivationStrength activationStrength;` ?

File: schemas/IDLs/org.gel.models.report.avro/6.1.0/CommonInterpreted.avdl (hunk 1 above)
Comment (Antonior26) [reply schemas/IDLs/org.gel.models.report.avro/6.1.0/CommonInterpreted.avdl]: thanks

File: schemas/IDLs/org.gel.models.report.avro/6.1.0/CommonInterpreted.avdl (hunk 2 above)
Comment (Antonior26) [reply schemas/IDLs/org.gel.models.report.avro/6.1.0/CommonInterpreted.avdl]: no first one is the type (enumeration) second is the name (camel case starting with lowercase)

**Pull request merged**